packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -766,6 +839,39 @@ export interface RollingUpdateConfiguration {
   readonly suspendProcesses?: ScalingProcess[];
 }
 
+/**
+ * A group of notificationTypes, you can use one of the predefined groups, such as NotificationTypes.ERRORS
+ * or create a custome group by instantiating a `NotificationTypes` object, e.g: `new NotificationTypes(`NotificationType.INSTANCE_LAUNCH`)`.
+ */
+export class NotificationTypes {
+  /**
+   * Fleet scaling erros
+   */
+  public static readonly ERRORS = new NotificationTypes(NotificationType.INSTANCE_LAUNCH_ERROR, NotificationType.INSTANCE_TERMINATE_ERROR);
+
+  /**
+   * All fleet scaling events
+   */
+  public static readonly ALL = new NotificationTypes(NotificationType.INSTANCE_LAUNCH,
+    NotificationType.INSTANCE_LAUNCH_ERROR,
+    NotificationType.INSTANCE_TERMINATE,
+    NotificationType.INSTANCE_TERMINATE_ERROR);
+
+  /**
+   * Fleet scaling launch events
+   */
+  public static readonly LAUNCH_EVENTS = new NotificationTypes(NotificationType.INSTANCE_LAUNCH, NotificationType.INSTANCE_LAUNCH_ERROR);
+
```

Comment:
any reason we wouldn't want a predefined type for `TERMINATE_EVENTS` as well? I can envision value in being able to subscribe fleet terminations to a topic

Reply:
leaving bread crumbs :)